File: INICIANTE/JAVASCRIPT/1046.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(texto) { return lines.shift(); };

var [a, b] = prompt("pontos").split(" ").map(Number)

if(b > a){
    console.log("O JOGO DUROU " + (b - a) + " HORA(S)")
}
else if(a == b){
    console.log("O JOGO DUROU 24 HORA(S)")
}
else if(a > b){
    console.log("O JOGO DUROU " + (24 - a + b) + " HORA(S)")
}

File: INICIANTE/JAVA/1046.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int a, b;
        Scanner sc = new Scanner(System.in);
        a = sc.nextInt();
        b = sc.nextInt();

        if(b > a){
            System.out.println("O JOGO DUROU " + (b - a) + " HORA(S)");
        }
        else if(a == b){
            System.out.println("O JOGO DUROU 24 HORA(S)");
        }
        else if(a > b){
            System.out.println("O JOGO DUROU " + (24 - a + b) + " HORA(S)");
        }
        sc.close();
    }
}
